sw/device/lib/dif/dif_aes_unittest.cc

```diff
@@ -171,15 +186,16 @@ class CTRTest : public AesTestInitialized {
 
 TEST_F(CTRTest, start) {
   EXPECT_READ32(AES_STATUS_REG_OFFSET, 1);
-  SetExpectedConfig(AES_CTRL_SHADOWED_KEY_LEN_VALUE_AES_128,
-                    AES_CTRL_SHADOWED_MODE_VALUE_AES_CTR,
-                    AES_CTRL_SHADOWED_OPERATION_VALUE_AES_ENC,
-                    /*manual_op=*/false,
-                    /*force_zero_masks=*/false);
+  SetExpectedConfig({.key_len = AES_CTRL_SHADOWED_KEY_LEN_VALUE_AES_128,
+                     .mode = AES_CTRL_SHADOWED_MODE_VALUE_AES_CTR,
+                     .operation = AES_CTRL_SHADOWED_OPERATION_VALUE_AES_ENC,
+                     .manual_op = false,
+                     .force_zero_masks = false,
+                     .key_sideloaded = false});
```

Comment:
```
{.key_len = AES_CTRL_SHADOWED_KEY_LEN_VALUE_AES_128,
                     .mode = AES_CTRL_SHADOWED_MODE_VALUE_AES_CTR,
                     .operation = AES_CTRL_SHADOWED_OPERATION_VALUE_AES_ENC,
                     .manual_op = false,
                     .force_zero_masks = false,
                     .key_sideloaded = false}
```

Maybe worth defining a default config, and then changing individual bits in specific test cases? Otherwise it seems that most of the time only one field changes, but we have to type out and initialise the entire structure every time.

Reply:
That was the original approach, but there were arguments that this way improves readability.